### ISSUE TITLE
Maintenance dashboard UI for collaborative data quality tasks, reports for monitoring progress

### ISSUE THREAD

**Issue by brierjon** [internetarchive/openlibrary#7630]
There is much potential in the data needs for OpenLibrary to pair up existing tools as well as bot and humans for content curation and data quality review. Currently most of this either occur in specialized tools not linked or easily discoverable from the OpenLibrary UI, documentation, or navigation and these are isolated from the content for the issues they are addressing as the issues are not exposed on OpenLibrary to prompt users. This could potentially be called "FullyBooked" or some library pun regarding curation and/or reaching high data quality, but names aside the function is what matters.

### Describe the problem that you'd like solved
Primarily helping the community coordinate on large scale tasks that need human in the loop resolution where bots alone cannot resolve.

Create a UI for aggregation of crowdsourcing review and data cleanup tasks to be shared and process per each task to be laid out in a clear format and workflow to afford users of any skill levels to join in reviewing content.

Critical to the design consideration is focusing on defined tasks needing human review, clear resolution workflows, and protocols to assess tasks were resolved correctly instead of creating large scale data quality issues.  
1. Defining known issues and listing them for task design - identify how the data is being addressed (fully automated if so, note the bot/script/quality process in place and who is the contact monitoring this vs task needing human in the loop) For human in the loop tasks who is the contact and what is the scale and need (one time vs ongoing) and if large
2. Define a workflow for each task and how the issue would be considered fix including defining how to flag false positives and notifying the task maintainer to improve the flagging or exempt from future tasks.
3. Designing an interface to discover one issue to check how the user marks complete to move on to the next tasked item to check.
4. Design to enable review tools for tracking what has been fixed by which users in case of large scale data issues present it could be tracked down to users of this tool.

The goal is for OpenLibrary to make the process of finding tasks more straightforward and enable more users to join in fixing of large scale issues by creating more "paint by number" oriented tasks thank needing to have internalized all the ways to fix issues. As well as creating a common area for those working on data quality to be able to add new quality review tools in a commons area for others to be aware of who and how they issues are being addressed.

(optional enhancement) Bring together existing tools as part of the clearing house architecture that are applicable to curation activities to make discovery of the tools more accessible from the site navigation and not buried in documentation.

### Proposal & Constraints
A possible integrated guiding concept for crowdsourcing task exposure in the OpenLibrary UI and complementary platform is the OpenFoodFacts Hunger Games site https://github.com/openfoodfacts/hunger-games https://hunger.openfoodfacts.org. Their interfaces pairs computationally flagged (image recognition tasks) as possible additions as suggested could be added to OpenFoodFacts both in the stand alone UI as well as on OpenFoodFacts product listing. Similarly OpenLibrary could use OCR and Image Recognition, Natural Language Processing, and basic regular expressions (like requested in #7629) to flag content to be added to fields on works/editions/authors, and build matching and data verification processes as tasks where human in the loop is critical to determine the difference.

This approach could be similarly applied to OpenLibrary in combination with feature request #7627 to rely on flags generated by existing and new bots as well as humans to identify what content should be reviewed, for what reason, and what is the process to resolve such data quality issue. ie document the cases and how to resolve them in a workflow and to identify edge cases not addressed. 

Alternative crowdsourcing task design could look toward Zooniverse.org as a broad design concept of crowdsourcing and guided task design, but the scope of Hunger Games of OpenFoodFacts seems more applicable in the tighter integration with a specific set of data management components. The most important aspect of Zooniverse is the ability to discuss specific tasks if resolution it is unclear or conflated with additional issues. 

A tool for OpenStreetMap may also be another tool for inspiration - Maproulette https://maproulette.org/ which functions as a collaborative QA task manager for which users can create and join in the task design and processing to resolve the issue or flag as "too hard" or "already fixed".

Designing of a UI framework that is flexible to display a variety of task types and workflows to process issue flags or labels for different content review. The task design should include a description of the problem, how to resolve, ways to flag edge cases that are not addressed in the "ways to resolve" to improve the documentation, examples of false positives, and workflow of resolving the task. Metrics for activities should be considered such as "what qualifies as a task" how to handle reverting users who don't understand the task or the task wasn't clear to track what changes might need to be reviewed that were performed via the task. 

Design of task and maintenance dashboard as a component of the progress tracking. For each task a scope of the issue to the best of the ability could be determined report the number of items queued and number completed at the minimum.

Ensure edits made from the UI banner and from the tool are tagged in the changelog to identify the maintenance tool as the interface and/or process to track in the recent changes as a facet and in review of changes diff https://openlibrary.org/recentchanges

### Additional context
Hunger Games
![image](https://user-images.githubusercontent.com/717866/224364533-18934180-99e7-480f-a7c1-1db1890f166d.png)

Maproulette
![image](https://user-images.githubusercontent.com/717866/224364103-a79508fd-af45-49a1-9134-b84de3b988fb.png)

Wikivoyage maintenance dashboard (as well as many other open collaborative projects list their tasks outstanding to encourage people to jump in an assist with the maintenance and data. Example: https://en.wikivoyage.org/wiki/Wikivoyage:Maintenance_panel as well as some WikiProjects have dashboards for task - https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Lakes/backlog

**Comment by tfmorris** [comment]
This is a cool idea, but a non-trivial implementation task. It might be worth starting with some super simple use cases like "Flag as SPAM" + "Vote on whether flagged item is SPAM" to build some experience and experiment.

So called "data games" were all the rage 10-15 years ago, so there's a fair amount of literature about them. In addition to the open source examples, there are commercial examples like Amazon Mechanical Turk or Crowdflower, which can be looked at for templates. Wikidata also has a number of games such as [Mix 'n Match](https://meta.wikimedia.org/wiki/Mix%27n%27match) which aren't mentioned above.

Freebase had a pretty sophisticated set of [data games](https://web.archive.org/web/20110903103202/http://wiki.freebase.com/wiki/Data_games) and supporting [infrastructure](https://web.archive.org/web/20110902113940/http://wiki.freebase.com/wiki/RABJ). One of the main researchers behind it, Praveen Paritosh, has published a number of [papers](https://research.google/people/PraveenParitosh/).

**Comment by brierjon** [comment]
@tfmorris I agree it is not a trivial implementation and that it should be focused a limited scope. I'm aware of the "data games" research as it intersected with my my PhD research in citizen science infrastructure and data. I was trying to drive the suggestion to focus on data objectives for the platform that align with the intrinsic motivations for editors and making an interface focused on completing the maintenance tasks. I was trying to avoid suggestions of gamification in the terms of these tasks.

I selected these examples as they are focused on functional elements that are useful for OpenLibrary focused on an interface focused on library needs for coordination and data accuracy ie more focused on expert user and  professional tools than gamification since the intrinsic value of quality data is there as the motivation.

**Comment by mekarpeles** [comment]
I agree with @tfmorris. This issue paints a nice picture for a world we may want to live in. I am marking it as a proposal topic for community discussion. It will likely be closed in favor of a more detailed `epic`, e.g. see: https://docs.google.com/document/d/1a8TAFERRxsfvgh8xHHqXjr2yC2-67Svo5TE4Z19B75Q/edit#heading=h.r0lvvv48gqdu

**Comment by tfmorris** [comment]
I understood the original @brierjon proposal to include a framework for actions, not strictly reporting.

@mekarpeles I don't see any quality metrics on that page at all. Far from "improving over time," data quality has plummeted during the period covered by the tall yellow spikes to the right of your graph (ie the last 10-12 months). The number of duplicate author records, conflated author records, contributors with roles (editor, illustrator, translator) embedded in the name string, etc has absolutely skyrocketed. 

The overwhelming majority of these are imported from Better World Books (BWB) with the rest coming from Amazon. Many of these are public domain books being copyright washed by junk POD mills -- a practice which seems antithetical to the goals of OpenLibrary.

In the context of book imports, data quality metrics should be driving whether or not a given batch of metadata is of high enough quality to import and, longer term, whether a given source is high enough quality to continue considering for use.
